src/environments.js

```diff
@@ -66,10 +66,12 @@ var environmentDefinitions = [
             var parser = this;
             // Currently only supports alignment, no separators like | yet.
             colalign = colalign.value.map ? colalign.value : [colalign];
-            colalign = colalign.map(function(node) {
+            var cols = colalign.map(function(node) {
                 var ca = node.value;
                 if ("lcr".indexOf(ca) !== -1) {
-                    return ca;
+                    return {
+                        align: ca
+                    };
```

Comment:
Having an `index` around makes it easier to link entries from the `colSpec` back to the actual content from the `body`. Sure, one could have that index in some other variable, and increment that. It's more a matter of convenience.